grafana/dashboards/events_withfields.json

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
Please move these 2 files to a subdirectory and place a readme explaining what is the difference between these and the original files.